tsdoc/schemas/tsdoc.schema.json

```diff
@@ -29,6 +29,20 @@
       }
     },
 
+    "supportedHtmlElements": {
+      "description": "The HTML element names that are supported in this configuration. Used in conjunction with the \"reportUnsupportedHtmlElements\" setting.",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "pattern": "^[a-zA-Z0-9]+$"
```

Comment:
TSDoc allows HTML5 custom elements, whose names have hyphens (e.g. `<article-info id="123" />`)

But I wouldn't recommend to try to capture the exact syntax in the regexp, since the JSON Schema engine's error messages will be more difficult for users to understand.  Allowing `-` is probably good enough here.

Reply:
fixed